### PR TITLE
Update .gitignore to include required var files

### PR DESCRIPTION
**Pull request by LRotherfield**
Currently if you reinit git in this project then all of the contents for /var/ are ignored so you end up missing required files such as OroRequirements.php

### CODE REVIEW

**Comment by orocla** [comment]
Thank you for your pull request.

It looks like this may be your first contribution to an Oro, Inc. open source project, in which case you'll need to sign a Contributor License Agreement (CLA) at http://www.orocrm.com/contributor-license-agreement/

If you've already signed the CLA, it's possible we don't have your GitHub username or you're using a different email address. GitHub uses the email address you set in your local Git configuration to associate commits with your GitHub account. Please sign the CLA again using the correct GitHub username and email address or see this help article on [setting the email on your git commits](https://help.github.com/articles/setting-your-email-in-git/).

Once you've signed the CLA, please allow for some time for the submission to be processed.

**Comment by mbessolov** [comment]
@LRotherfield what is the correct spelling of your GitHub username? I see that you've signed our CLA a few times already using "lrotherfield" (all lowercase), while it seems that when our robot is checking the author of the commits, GitHub returns "LRotherfield", same as when I search you on GitHub.

File: .gitignore [hunk 1]
```diff
@@ -5,11 +5,22 @@
 /node_modules
 /composer.phar
 behat.yml
-/var/
+/var/*
+!var/cache/
+/var/cache/*
 !var/cache/.gitkeep
```
Comment (anyt) [inline .gitignore]: There is no need to add
```
!var/cache/
/var/cache/*
```
as we have .gitkeep inside this folder that already in `.gitignore`, so folder will not be deleted after git init. 
Same for `logs`, `attachments` and` import_export` folders

**Comment by LRotherfield** [comment]
@mbessolov my username is LRotherfield https://github.com/LRotherfield

File: .gitignore (hunk 1 above)
Comment (LRotherfield) [reply .gitignore]: @anyt if you have the following rules the var/cache/.gitkeep file will not be added to a new repository:
```
/var/* 
!var/cache/.gitkeep
```
With or without the `*` above makes no difference

Hopefully I am not doing something silly, but this post seems to explain it: https://stackoverflow.com/a/5534865/358202

Comment (LRotherfield) [reply .gitignore]: I guess if you know which folders you want to ignore in `var` then we could have far less rules.  i.e if you only want to ignore cache, logs, attachments and import_export then you only need 8 rules in pairs like:
```
/var/cache/*
 !var/cache/.gitkeep
```
That would save a rule per directory compared to my patch

Comment (LRotherfield) [reply .gitignore]: @anyt did you see my above comments ^?

**Comment by LRotherfield** [comment]
@anyt did you see my responses to your change request?

**Comment by anyt** [comment]
Thanks for the reminder, @LRotherfield, I thought that already answered to this issue. My fault.
I already checked your changes, you are right. 
 Approved. 
I'll merge this till the end of the week. 
Need to port them to all other applications as well.

**Comment by anyt** [comment]
Merged, https://github.com/oroinc/platform-application/commit/f96c1eccca03a9938611f9df1bdb6b84254ed0c6.

Thank you for contribution, @LRotherfield !

**Pull request closed without merging**